tests/src/Mod/Sketcher/App/SketcherTestHelpers.cpp

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+#include <App/Application.h>
+#include <App/Document.h>
+#include <App/Expression.h>
+#include <App/ObjectIdentifier.h>
+#include <Mod/Sketcher/App/GeoEnum.h>
+#include <Mod/Sketcher/App/SketchObject.h>
+#include "SketcherTestHelpers.h"
+
+void SketchObjectTest::SetUpTestSuite()
+{
+    tests::initApplication();
+}
+
+void SketchObjectTest::SetUp()
+{
+    _docName = App::GetApplication().getUniqueDocumentName("test");
+    auto _doc = App::GetApplication().newDocument(_docName.c_str(), "testUser");
+    // TODO: Do we add a body newName, or is just adding sketch sufficient for this test?
+    _sketchobj = static_cast<Sketcher::SketchObject*>(_doc->addObject("Sketcher::SketchObject"));
```

Comment:
```suggestion
    _sketchobj = _doc->addObject<Sketcher::SketchObject>();
```

Reply:
Could you point me to the function declaration/definition? It doesn't seem to be there in `main` yet.

Reply:
I'm sorry it in here: https://github.com/FreeCAD/FreeCAD/pull/19142
It's what I'm working on, and not really ready yet.

Ignore this suggestion >_<